openfisca_us/parameters/usda/snap/asset_test/limit.yaml

```
@@ -0,0 +1,32 @@
+description: Maximum value of SNAP-visible assets
+standard:
+  description: Maximum value of SNAP-visible assets for households without elderly or disabled members
+  values:
+    2020-10-01:
+      value: 2_250
+      reference:
+        - title: SNAP - Fiscal Year 2021 Cost-of-Living Adjustments
+          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/COLAMemoFY2021.pdf
+    2021-10-01: 
+      value: 2_500
+      reference:
+        - title: SNAP - Fiscal Year 2022 Cost-of-Living Adjustments
+          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/2022-SNAP-COLA-%20Maximum-Allotments.pdf
+elderly_disabled:
+  description: Maximum value of SNAP-visible assets for households with elderly or disabled members
+  values:
+    2010-10-01:
```

At least per the source
```suggestion
    2020-10-01:
```